src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -184,7 +189,27 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
                 catch (const std::exception& e)
                 {
                     logError(WM_VULNSCAN_LOGTAG, "Error processing message: %s", e.what());
-                    // TO DO -> call to reinitialize the database.
+
+                    m_descriptionsDatabase->deleteAll();
+                    m_remediationsDatabase->deleteAll();
+                    m_translationsDatabase->deleteAll();
+                    m_cvesDatabase->deleteAll();
+
+                    for (auto& [name, database] : m_feedsDatabases)
+                    {
+                        database->deleteAll();
+                    }
+
+                    const int offset = 0;
+                    const std::string url = "localhost/ondemand/" + topic_name + "?offset=" + std::to_string(offset);
```

Comment:
I think we can make these lines more simple, changing to

```cpp
const std::string url = "localhost/ondemand/" + topic_name + "?offset=0";
```

And avoid a call of `std::to_string()`. What do you think about it?

Reply:
Done